featuretests/cmd_juju_updateseries_test.go

```diff
@@ -1,37 +0,0 @@
-// Copyright 2017 Canonical Ltd.
-// Licensed under the AGPLv3, see LICENCE file for details.
-
-package featuretests
-
-import (
-	jc "github.com/juju/testing/checkers"
-	gc "gopkg.in/check.v1"
-
-	jujutesting "github.com/juju/juju/juju/testing"
-	"github.com/juju/juju/testing/factory"
-)
-
-type cmdUpdateSeriesSuite struct {
-	jujutesting.JujuConnSuite
-}
-
-func (s *cmdUpdateSeriesSuite) TestUpdateApplicationSeries(c *gc.C) {
```

Comment:
Should we change this one to test the replacement?  juju set-series for application?

Reply:
This appears to be tested in _cmd_juju_setseries_test.go_.